ocs_ci/helpers/helpers.py

```diff
@@ -14,9 +14,11 @@
 import threading
 import time
 import inspect
+import platform
```

Comment:
As general note, if odf_cli is new topic, i would prefer start new module instead using a general helper file (that already too large)